platformio.ini

```diff
@@ -271,6 +271,31 @@ lib_ignore    = U8glib-HAL
 lib_ldf_mode  = 1
 src_filter    = ${common.default_src_filter} +<src/HAL/HAL_STM32F1>
 monitor_speed = 250000
+#
+# BIGTREE_SKR_MINI
+#
+[env:BIGTREE_SKR_MINI]
+platform      = ststm32@<4.4.0
```

Comment:
any reason for this <4.4.0 ?

Reply:
Do you think we even need this new env section?

Side note: I'm pretty sure each `env` can inherit from any other `env`, so if there are going to be multiple entries for a particular platform, we should make one 'root' `env` for the platform with all the common options (e.g., `common_STM32F1`) and then just include the differences in the leaf `env`s.

Reply:
I found no full environment inheritance option. But it seems you can use individual field values from other environments. Not a big help in reducing `platformio.ini` size.

Reply:
i think its required for the STM32F1.py .. i guess they use a bootloader for the upgrade via sdcard...

Reply:
@thinkyhead  Don't have to add new env section, increase in STM32f1 env: extra_scripts = buildroot/share/PlatformIO/scripts/STM32f1 py.The file name of stm32f1.py can be changed according to the user's needs. Basically, STM32F103xxx can be grouped into a series, which are all the same. The others are the model of a new board and the pin file of the board .

Reply:
I have not had the time to make these changes. Also, I don't feel qualified.